tutorials/building_a_simple_web_blog_with_vweb/README.md

```diff
@@ -372,12 +363,24 @@ pub fn (mut app App) new_article() vweb.Result {
 > Untyped `form['key']` is temporary. Very soon Vweb will accept query and form
 parameters via function arguments: `new_article(title, text string) {`.
 
+The decorator on our function tells vweb the path to our endpoint, `/new_article`,
+and that it is an HTTP POST type operation.
+
 We need to update `index.html` to add a link to the "new article" page:
 
 ```html
 <a href='/new'>New article</a>
 ```
 
+Next we need to add the HTML endpoint to our code like we did with `index.html`:
+
+```v ignore
+pub fn (mut app App) new() vweb.Result {
```

Comment:
New to add `[/new]` route attribute.  A change is going in that will make you have to have explicit route attributes for the endpoint to be exposed.

Reply:
Updated in README.md and the source code.